lib/bike_brigade_web/live/campaign_signup_live/show.html.heex

```diff
@@ -1,56 +1,74 @@
-<.header>
-  <%= @campaign.program.name %> - <%= campaign_date(@campaign) %>
+<.header class="text-center mt-4 mb-8 md:text-left">
+  <div class="hidden md:flex">
+    <%= @campaign.program.name %> - <%= campaign_date(@campaign) %>
+  </div>
+  <div class="text-xl flex flex-col mb-4 md:hidden"><%= @campaign.program.name %></div>
+  <div class="text-sm mt-2 md:hidden">Deliveries for <%= campaign_date(@campaign) %></div>
   <:subtitle>
-    <span class="font-bold"><%= @campaign.location.address %></span>
-    <span :if={@campaign.program.campaign_blurb}>
-      - <%= @campaign.program.campaign_blurb %>
-    </span>
+    <span class="text-sm font-bold">Pickup: <%= @campaign.location.address %></span>
+    <span :if={@campaign.program.campaign_blurb}>- <%= @campaign.program.campaign_blurb %></span>
   </:subtitle>
 </.header>
 
-<section>
-  <.table id="tasks" rows={@tasks}>
+<% sorted_tasks = Enum.sort_by(@tasks, & &1.id) %>
```

Comment:
Is this just to make the tasks sorting consistent across page loads?

I think we can fix the query to sort the tasks inside of `campaign_riders_and_tasks`. WDYT?